mobile_verifier/migrations/33_data_sets.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TYPE data_set_status AS enum (
+       'pending',
+       'downloaded',
+       'processed'
+);
+
+CREATE TYPE data_set_type AS enum (
+       'urbanization',
+       'footfall',
+       'landtype'
+);
+
+CREATE TABLE data_sets (
```

Comment:
Can we use a more descriptive name than `data_sets`?
If you're looking only at the database it tells you very little about which modules would use this table, and what type of interactions might be happening to it.

Maybe `hex_boosting_data_set_file_downloading`?

It tells you what type of data it's concerned about. That it's not storing an actual data set, but the status of a file. And the file is being downloaded from somewhere else.

Reply:
maybe `hex_assignment_data_set_status`?? or maybe plural?

Reply:
I like that better :+1:

Reply:
done